From 8ca0b6c9dd95ee37579bbf9897b001e2397aafb5 Mon Sep 17 00:00:00 2001
From: Jose Lamego <jose.a.lamego@intel.com>
Date: Tue, 21 Aug 2018 10:01:39 -0500
Subject: [PATCH kubernetes] kubelet.service: fix crio.sock path

container-runtime-endpoint is targeted to an outdated path for
crio.sock. This change updates the path to /var/run/crio directory
where it is delivered by CRI-O

Fix: https://github.com/clearlinux/distribution/issues/158

Signed-off-by: Jose Lamego <jose.a.lamego@intel.com>
---
 kubelet.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kubelet.service b/kubelet.service
index 5d06a0e..17ae4ef 100644
--- a/kubelet.service
+++ b/kubelet.service
@@ -9,7 +9,7 @@ Environment="KUBELET_NETWORK_ARGS=--network-plugin=cni --cni-conf-dir=/etc/cni/n
 Environment="KUBELET_DNS_ARGS=--cluster-dns=10.96.0.10 --cluster-domain=cluster.local"
 Environment="KUBELET_AUTHZ_ARGS=--authorization-mode=Webhook --client-ca-file=/etc/kubernetes/pki/ca.crt"
 Environment="KUBELET_CADVISOR_ARGS=--cadvisor-port=0"
-Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --container-runtime-endpoint=unix:///var/run/crio.sock --runtime-request-timeout=30m"
+Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --container-runtime-endpoint=unix:///var/run/crio/crio.sock --runtime-request-timeout=30m"
 ExecStart=/usr/bin/kubelet $KUBELET_KUBECONFIG_ARGS $KUBELET_SYSTEM_PODS_ARGS $KUBELET_NETWORK_ARGS $KUBELET_DNS_ARGS $KUBELET_AUTHZ_ARGS $KUBELET_CADVISOR_ARGS $KUBELET_EXTRA_ARGS
 Restart=always
 StartLimitInterval=0
-- 
2.18.0

